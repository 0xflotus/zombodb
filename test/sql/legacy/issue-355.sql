CREATE TABLE issue355 (
   id serial8,
   name text,
   label zdb.phrase,
   description zdb.fulltext
);

CREATE INDEX idxissue355 ON issue355 USING zombodb( (issue355.*) ) WITH (field_lists='name=[name, label, description]');

SELECT * FROM zdb.tally('idxissue355', 'name', '^.*', '', 10, 'reverse_count');

DROP TABLE issue355;