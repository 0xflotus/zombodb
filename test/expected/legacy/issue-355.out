CREATE TABLE issue355 (
   id serial8,
   name text,
   label phrase,
   description fulltext
);
CREATE INDEX idxissue355 ON issue355 USING zombodb(zdb('issue355', ctid), zdb(issue355)) WITH (field_lists='name=[name, label, description]', url='localhost:9200/');
SELECT * FROM zdb_tally('issue355', 'name', '^.*', '', 10, 'count');
 term | count 
------+-------
(0 rows)

DROP TABLE issue355;
